### ISSUE TITLE
images should have alt text

### ISSUE THREAD

**Issue by discospiff** [TripleDavid/IS7012#4]
-It's a great idea to add alt text (using the alt attribute) to images.  This makes them friendly for screen readers.  The text should describe the image in a concise manner, so that someone who can't see the image will know what it represents.

https://github.com/TripleDavid/IS7012/blob/c346837145fd76882f4dde067954a91b75c089f5/Week%201/dickendd-about-me-styled.html#L85
